flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/stream/sql/CalcITCase.scala

```diff
@@ -548,4 +549,24 @@ class CalcITCase extends StreamingTestBase {
     TestBaseUtils.compareResultAsText(result, "42")
   }
 
+  @Test
```

Comment:
I think it would be good to find a better place for the TableDescriptor-related tests. Acknowledging that they are expensive, can we create a TableDescriptorITCase if we get rid of another one? There seem to be some with a single test and no additional setup that could easily be transferred into `CalcITCase`.

Reply:
I agree. Having one ITCase for descriptors was ok in the last PR. But we should not add more. We can extend `org.apache.flink.table.api.TableEnvironmentTest` and check what has been put into the in-memory catalog.

Reply:
Sounds good. The only test case in there will be removed with this FLIP as well anyway. I'll also add one for createTemporaryTable so we have all APIs tested in there, plus the one IT case.